### PR TITLE
Fix docs

### PR DESCRIPTION
**Pull request by prxgr4mm3r**

### CODE REVIEW

File: docs/docs/smart-contracts/example/data.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ impl openbrush::traits::OccupyStorage for PointData {
 }
 ```
 
-Also, you can use the `openbrush::upgradeable_storage` macro that implements that trait by default,
+Also, you can use the `openbrush::storage_item` macro that implements that trait by default,
```
Comment (coreggon11) [inline docs/docs/smart-contracts/example/data.md:71]: this whole file needsto be reworked

**Review by coreggon11 — CHANGES_REQUESTED** [review]

File: docs/docs/smart-contracts/PSP34/Extensions/mintable.md [hunk 2]
```diff
@@ -7,17 +7,33 @@ This example shows how you can reuse the implementation of [PSP34](https://githu
...
+#![cfg_attr(not(feature = "std"), no_std, no_main)]
+
+#[openbrush::implementation(PSP34, PSP34Mintable)]
+#[openbrush::contract]
+pub mod my_psp34_mintable {
+    use openbrush::traits::Storage;
+
+    #[derive(Default, Storage)]
+    #[ink(storage)]
+    pub struct Contract {
```
Comment (Artemka374) [inline docs/docs/smart-contracts/PSP34/Extensions/mintable.md:24]: I'm not sure, but as I remember, there should be a field for psp34::metadata as well

File: docs/docs/smart-contracts/example/contract.md [hunk 3]
```diff
@@ -21,15 +21,15 @@ authors = ["Brushfam <dominik.krizo@727.ventures>"]
...
+ink = { version = "4.2.1", default-features = false}
 scale = { package = "parity-scale-codec", version = "3", default-features = false, features = ["derive"] }
 scale-info = { version = "2.6", default-features = false, features = ["derive"], optional = true }
 
 # These dependencies
 shares_contract = { path = "../shares", default-features = false, features = ["ink-as-dependency"]  }
 loan_contract = { path = "../loan", default-features = false, features = ["ink-as-dependency"]  }
 lending_project = { path = "../..", default-features = false }
-openbrush = { version = "~3.1.1", default-features = false, features = ["pausable", "access_control"] }
+openbrush = { git = "https://github.com/Brushfam/openbrush-contracts", branch = "feature/stable-rust", default-features = false }
```
Comment (Artemka374) [inline docs/docs/smart-contracts/example/contract.md:32]: it's better to change feature/stable-rust here to develop or main

**Review by Artemka374 — CHANGES_REQUESTED** [review]

File: docs/docs/smart-contracts/example/contract.md (hunk 3 above)
Comment (coreggon11) [reply docs/docs/smart-contracts/example/contract.md]: for now I used this branch everywhere so docs are aligned with code when testing

File: docs/docs/smart-contracts/PSP34/Extensions/mintable.md (hunk 2 above)
Comment (coreggon11) [reply docs/docs/smart-contracts/PSP34/Extensions/mintable.md:24]: i think that is not necessary in mintable example

Comment (Artemka374) [reply docs/docs/smart-contracts/PSP34/Extensions/mintable.md:24]: yeah, i was thinking that it's metadata, sry

**Review by coreggon11 — COMMENTED** [review]
lgtm, just some spelling comments

**Pull request merged**